circuit/src/androidMain/kotlin/com/slack/circuit/Navigator.android.kt

```diff
@@ -44,7 +52,11 @@ internal class NavigatorImpl(
   override fun goTo(screen: Screen) = backstack.push(screen)
 
   override fun pop(): Screen? {
-    check(!backstack.isAtRoot) { "Cannot pop the root screen." }
+    if (backstack.isAtRoot) {
```

Comment:
Note: this will only ever be invoked by manual calls to Navigator.pop from within the presenter. This is because the custom back handler we configure in `NavigableCircuitContent` is only enabled when the backstack.size > 1

Reply:
This general approach still allows the project to configure its own overriding back handler if they don't want to pop the host activity